bower.json

```diff
@@ -33,7 +31,12 @@
     "webapp-module-namespace": "https://github.com/wireapp/webapp-module-namespace.git#1.0.0",
     "webrtc-adapter": "https://github.com/webrtc/adapter.git#2.0.8",
     "wire-audio-files": "https://github.com/wireapp/wire-audio-files.git#1.1.0",
-    "wire-theme": "https://github.com/wireapp/wire-theme.git#1.0.0"
+    "wire-theme": "https://github.com/wireapp/wire-theme.git#1.0.0",
+    "wire-webapp-cryptobox": "https://github.com/wireapp/wire-webapp-cryptobox.git#2.0.5"
+  },
```

Comment:
We don't have an organisation on npm, so it's hard to get some names on npmjs.com. For example `proteus` was taken. So I thought it might be a good idea to prefix everything (what is created by our team) with `wire-webapp-`.